src/write_options.rs

```diff
@@ -0,0 +1,131 @@
+/// Options to control how Lofty writes to a file
+#[derive(Copy, Clone, Debug, Ord, PartialOrd, Eq, PartialEq)]
```

Comment:
Making this public type _copyable_ could require breaking API changes in the future if it no longer implements `Copy`. If not needed internally then better remove this derive. Could be added later when it is actually needed.

Reply:
And the `Ord` traits don't make any sense to me.

Reply:
Good catch with `Ord`. I just copied `ParseOptions` and renamed it to `WriteOptions`. Guess I copied it from some other struct. I'll have to remove it from both.

Reply:
> in the future if it no longer implements Copy

I really only intend for `WriteOptions` to be ints and bools just like `ParseOptions`. Since it's used in generated code in `lofty_attr`, it'll just be easier to keep it `Copy` for now.

I'm not too worried about breaking changes at the moment, especially since I have to work on #345 sometime in the near future.